Code/EnginePlugins/JoltPlugin/Character/Implementation/JoltCharacterControllerComponent.cpp

```diff
@@ -69,7 +69,7 @@ void ezJoltCharacterControllerComponent::SerializeComponent(ezWorldWriter& inout
 void ezJoltCharacterControllerComponent::DeserializeComponent(ezWorldReader& inout_stream)
 {
   SUPER::DeserializeComponent(inout_stream);
-  const ezUInt32 uiVersion = inout_stream.GetComponentTypeVersion(GetStaticRTTI());
+  const ezUInt32 uiVersion = inout_stream.GetComponentTypeVersion(GetStaticRTTI()); // NOLINT: ignore unused variable
```

Comment:
You could also just comment out these lines as long as the version isn't needed yet.